test/tempfile.cpp

```diff
@@ -42,18 +42,16 @@ namespace unittests
 TempFile::TempFile(const char* name)
 {
 #ifdef _WIN32
-  std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> utf8ToUtf16;
+  std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> utfConv;
   wchar_t cbase[MAX_PATH];
   wchar_t ctmp[MAX_PATH];
-  const std::wstring wname = utf8ToUtf16.from_bytes(name);
+  const std::wstring wname = utfConv.from_bytes(name);
   GetTempPathW(MAX_PATH-(wname.size()+2), cbase);
   // This create a file for us, ensure it is unique.
   // So we need to delete it and create the directory using the same name.
   GetTempFileNameW(cbase, wname.c_str(), 0, ctmp);
   fd_ = _wopen(ctmp, _O_CREAT | _O_TEMPORARY | _O_SHORT_LIVED | _O_RDWR | _O_TRUNC);
-  const int pathSize = WideCharToMultiByte(CP_UTF8, WC_ERR_INVALID_CHARS, ctmp, -1, NULL, 0, NULL, NULL);
-  path_.assign(pathSize, 0);
-  WideCharToMultiByte(CP_UTF8, WC_ERR_INVALID_CHARS, &ctmp[0], -1, &path_[0], pathSize, NULL, NULL);
+  path_ = utfConv.to_bytes(ctmp);
```

Comment:
Please, squash this commit directly with bb7c89859 instead of having a commit using `WideCharToMultiByte` and another removing in the same PR.

Reply:
Done